### PR TITLE
logo

### PR DESCRIPTION
**Pull request by solanki505**

### CODE REVIEW

File: Screenshot_2024-05-18_174852-removebg-preview.png
Comment (codervivek5) [inline Screenshot_2024-05-18_174852-removebg-preview.png:1]: can you add a background color only in circle use light color and make VigyBag bold and not seperate it also

**Review by codervivek5 — CHANGES_REQUESTED** [review]
try to make these changes then send me sample plz

File: Screenshot_2024-05-18_174852-removebg-preview.png
Comment (solanki505) [reply Screenshot_2024-05-18_174852-removebg-preview.png:1]: ok I will make this changes

Comment (codervivek5) [reply Screenshot_2024-05-18_174852-removebg-preview.png:1]: sure

File: Screenshot_2024-05-18_183654-removebg-preview.png
Comment (codervivek5) [inline Screenshot_2024-05-18_183654-removebg-preview.png:1]: make a copy of above design same like this also

File: New Project1.png
Comment (codervivek5) [inline New Project1.png:1]: @solanki505 border is looking osm. can you add it in all side ?

**Review by codervivek5 — CHANGES_REQUESTED** [review]

File: New Project1.png
Comment (solanki505) [reply New Project1.png:1]: > @solanki505 border is looking osm. can you add it in all side ?

okay

File: Screenshot_2024-05-18_183654-removebg-preview.png
Comment (solanki505) [reply Screenshot_2024-05-18_183654-removebg-preview.png:1]: done

Comment (solanki505) [reply Screenshot_2024-05-18_183654-removebg-preview.png:1]: @codervivek5  please suggest if further changes are required, if the changes are okay please assign the task #1067 to me.

**Comment by codervivek5** [comment]
kindly solve the merge conflit

**Comment by solanki505** [comment]
@codervivek5 

ok sir

**Comment by solanki505** [comment]
Sir does the changes are okay or it need further changes please suggest

**Comment by codervivek5** [comment]
make sure your all logos are separate in the root folder with name  "New Logo"

**Comment by solanki505** [comment]
> make sure your all logos are separate in the root folder with name "New Logo"

okay @codervivek5 sir

**Comment by codervivek5** [comment]
Have you done this ?

**Comment by solanki505** [comment]
> Have you done this ?

Yes I separated it in folder new logo.

**Pull request closed without merging**